clients/fides-js/src/components/tcf/TcfVendors.tsx

```diff
@@ -98,8 +98,8 @@ const DataCategories = ({
     return null;
   }
 
-  // @ts-ignore this type doesn't exist in v2.2 but does in v3
   const declarations: GvlDataDeclarations | undefined =
+    // @ts-ignore this type doesn't exist in v2.2 but does in v3
```

Comment:
unrelated to this PR, but moved the ts-ignore down since it's this line that needs it